plugins/modules/api.py

```diff
@@ -444,13 +489,13 @@ def api_query(self):
             keys[k] = Key(k)
         try:
             if self.where:
-                if self.where[1] == '==':
+                if self.where[1] == '==' or self.where[1] == 'eq':
```

Comment:
I would write this as:
```suggestion
                if self.where[1] in ('==', 'eq'):
```
(easier to read and change); same for the other ones below